src/simulation/m_global_parameters.fpp

```diff
@@ -982,6 +1047,9 @@ contains
         if (qbmm .and. .not. polytropic) then
             allocate (MPI_IO_DATA%view(1:sys_size + 2*nb*4))
             allocate (MPI_IO_DATA%var(1:sys_size + 2*nb*4))
+        elseif (lag_bubbles) then
```

Comment:
perhaps we should change the code logic/branching to bubbles -> "Euler" or "Lagrange" ->  (if euler [set options], else if lagrange [set options]). ultimately i'd like to see the 'bubbles' keyword be the first class term for sub-grid bubbles, and then we differentiate down from there.